Fix 1008 - Remove redundant app namespace on the login activity layout

somayaj opened the pull request:
Description
The bug fix was to remove the redundant app namespace on the login activity layout. That has been removed as a fix for the defect. 
Fixes # [[1008]](https://github.com/anitab-org/mentorship-android/issues/1008)

Type of Change:
Code
Bug fix (non-breaking change which fixes an issue)
How Has This Been Tested?
Verified the layout on rebuild, no downstream effects were seen. 

Checklist:
 My PR follows the style guidelines of this project
 I have performed a self-review of my own code or materials
 Code/Quality Assurance Only
 My changes generate no new warnings

somayaj committed:
Removed redundant namespace declartion for app - #1008

somayaj committed:
Description
The bug fix was to remove the redundant app namespace. That has been removed as a fix for the defect.
Fixes # [1008]

Type of Change:
Code
Bug fix (non-breaking change which fixes an issue)
How Has This Been Tested?
Verified the layout on rebuild, no downstreem effects were seen.

Checklist:
 My PR follows the style guidelines of this project
 I have performed a self-review of my own code or materials
 Code/Quality Assurance Only
 My changes generate no new warnings

Removed redundant namespace declaration for app - #1008

somayaj committed:
Description
The bug fix was to remove the redundant app namespace. That has been removed as a fix for the defect.
Fixes # [1008]

Type of Change:
Code
Bug fix (non-breaking change which fixes an issue)
How Has This Been Tested?
Verified the layout on rebuild, no downstreem effects were seen.

Checklist:
 My PR follows the style guidelines of this project
 I have performed a self-review of my own code or materials
 Code/Quality Assurance Only
 My changes generate no new warnings

somayaj commented:
Please merge the PR. It was assigned to me and required a new branch to be created, hence the multiple PRs,. Deleted one and left this one which has the fix. 

Thanks!

sanchi0204 commented:
pls mention the issue and the earlier PR that has been closed

sanchi0204 commented:
edit this line in PR desc
Fixes # [1008]

to make it a hyperlink

somayaj commented:
done

On Thu, Oct 15, 2020 at 4:40 AM Sanchi Rastogi <notifications@github.com>
wrote:

> edit this line in PR desc
> Fixes # [1008]
>
> to make it a hyperlink
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-709046392>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIRAZGL27CPYJXQXBNWVFE3SK27QNANCNFSM4SEA656A>
> .
>

somayaj commented:
done

On Thu, Oct 15, 2020 at 4:49 AM asha somayajula <somayajulaas@gmail.com>
wrote:

> done
>
> On Thu, Oct 15, 2020 at 4:40 AM Sanchi Rastogi <notifications@github.com>
> wrote:
>
>> edit this line in PR desc
>> Fixes # [1008]
>>
>> to make it a hyperlink
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-709046392>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AIRAZGL27CPYJXQXBNWVFE3SK27QNANCNFSM4SEA656A>
>> .
>>
>

somayaj commented:
Can this be merged now?

sanchi0204 commented:
What changes have you done? Since the files changed section shows 0 changes

somayaj commented:
The PR? was it merged?

On Tue, Oct 20, 2020 at 12:20 PM Sanchi Rastogi <notifications@github.com>
wrote:

> What changes have you done? Since the files changed section shows 0 changes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-713016326>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIRAZGNN7HF2Y2FZXMSSF4DSLXBF5ANCNFSM4SEA656A>
> .
>

vj-codes reviewed (changes requested):
@somayaj to review or merge the PR , no changes are seen . Can you please update the PR

devkapilbansal commented:
@vj-codes the changes requested were related to linking and editing PR description.
That's why there are no changes
Can be verified by going up some comments
https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-709046035

vj-codes commented:
@devkapilbansal ahh no what I meant was in the `Files changed ` section on top , do see it's `O `?
How will I review if I don't see any file that is changed 👀

devkapilbansal commented:
> @devkapilbansal ahh no what I meant was in the `Files changed ` section on top , do see it's `O `?
> How will I review if I don't see any file that is changed

Ohh my bad I thought you are talking about changes since last review

somayaj commented:
No worries

On Tue, Oct 20, 2020 at 3:16 PM Kapil Bansal <notifications@github.com>
wrote:

>
>
>
>
> @devkapilbansal <https://github.com/devkapilbansal> ahh no what I meant
> was in the Files changed section on top , do see it's O ?
>
>
> How will I review if I don't see any file that is changed
>
>
>
>
> Ohh my bad I thought you are talking about changes since last review
>
>
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-713113548>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIRAZGKKAJD77H5ZA4KXRVTSLXV2XANCNFSM4SEA656A>
> .
>
>
>

epicadk commented:
@somayaj No changes have been made to any files can you please fix this?

somayaj commented:
The PR has been updated with the changes. there are no code changes to be
made.

On Wed, Oct 28, 2020 at 2:09 AM Aditya Kurkure <notifications@github.com>
wrote:

> @somayaj <https://github.com/somayaj> No changes have been made to any
> files can you please fix this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/anitab-org/mentorship-android/pull/1013#issuecomment-717745899>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIRAZGM4TS4X4O2IGU3TPHLSM67SPANCNFSM4SEA656A>
> .
>

somayaj committed:
Removed redundant namspace declaration for app - #1008

devkapilbansal commented:
@somayaj please squash your commits into one. Also,  feel free to change your PR title to something more descriptive

somayaj committed:
Merge remote-tracking branch 'origin/fix-1008' into fix-1008

somayaj commented:
No longer needed.
